### DISCUSSION TITLE
Choice of assignees strangely limited if project item linked to issue

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#65394] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

For example, this one is draft and allows selection of any org member:

![image](https://github.com/community/community/assets/10906554/3c243658-6728-4b15-83f1-36ec84bd043c)

But this one is issue and is strangely limited to two:

![image](https://github.com/community/community/assets/10906554/274f7cc4-5923-48a4-a0fd-614b6cd7bb5d)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by adamlui** [comment]
The problematic project is at https://github.com/orgs/KudoAI/projects/1/views/1

**Comment by tygoegmond** [comment]
Individuals you wish to assign must have access to the repository where the issue is created.

**Reply by adamlui** [reply]
They do, but even if they didn't, think about the first shot. What beneficial reason is there to exclude certain members but not others from being assignable to a task, merely on the basis it is issue-related?
